add aws release 20.0.0 (#1204)
giantswarm/releases: aws/v20.0.0/announcement.md

> **Workload cluster release v20.0.0 for AWS is available**. The new major version is the first release supporting Kubernetes `v1.25`. Please remember that with Kubernetes `v1.25` the Pod Security Policies (PSP) are removed from the API, hence customer workloads should also be adjusted *BEFORE* the upgrade. From now on the Pod Security Standards (PSS) are the default for securing the clusters. This release will also be used as a base for Giant Swarm `Vintage` to Giant Swarm adoption of `Cluster API for AWS` (CAPA) migration. Further details can be found in the [release notes](https://docs.giantswarm.io/changes/workload-cluster-releases-aws/releases/aws-v20.0.0/).

Reviewer: ```suggestion
**Workload cluster release v20.0.0 for AWS is available**. The new major version is the first release supporting Kubernetes `v1.25`. Please remember that with Kubernetes `v1.25` the Pod Security Policies (PSP) are removed from the API, so all workloads must have their PSPs removed *BEFORE* starting the upgrade. From now, on the Pod Security Standards (PSS) are the default for securing the clusters. This release will also be used as a base for Giant Swarm `Vintage` to Giant Swarm adoption of `Cluster API for AWS` (CAPA) migration. Further details can be found in the [release notes](https://docs.giantswarm.io/changes/workload-cluster-releases-aws/releases/aws-v20.0.0/).
```

> **Workload cluster release v20.0.0 for AWS is available**. The new major version is the first release supporting Kubernetes `v1.25`. Please remember that with Kubernetes `v1.25` the Pod Security Policies (PSP) are removed from the API, so all workloads must have their PSPs removed *BEFORE* starting the upgrade. From now, on the Pod Security Standards (PSS) are the default for securing the clusters. This release will also be used as a base for Giant Swarm `Vintage` to Giant Swarm adoption of `Cluster API for AWS` (CAPA) migration. Further details can be found in the [release notes](https://docs.giantswarm.io/changes/workload-cluster-releases-aws/releases/aws-v20.0.0/).

Reviewer: Sorry, I forgot to change this in both places
```suggestion
**Workload cluster release v20.0.0 for AWS is available**. The new major version is the first release supporting Kubernetes `v1.25`. Please remember that with Kubernetes `v1.25` the Pod Security Policies (PSP) are removed from the API, so all workloads must have their PSPs removed *BEFORE* starting the upgrade. From now, on the Pod Security Standards (PSS) are the default for securing the clusters. This release will also be used as the base for the migration from `Giant Swarm Vintage` to `Cluster API for AWS` (CAPA). Further details can be found in the [release notes](https://docs.giantswarm.io/changes/workload-cluster-releases-aws/releases/aws-v20.0.0/).
```
